### DISCUSSION TITLE
deepflow监控物理机跟docker运行的server对接咨询

### DISCUSSION THREAD

**Discussion by zcl1115** [deepflowio/deepflow#5741] [General]
目前我按照https://deepflow.io/docs/zh/ce-install/all-in-one/ 这个文档中介绍的Docker Compose搭建了deepflow的server，但是在搭建deepflow-agent的时候看https://deepflow.io/docs/zh/ce-install/legacy-host/ 这个文档里面没有涉及到docker的应该怎么操作，都是k8s，想问问能不能更新下这个监控传统服务器的文档

**Comment by 1473371932** [comment]
下面有呀，看部署 deepflow-agent 那片文档的第四步，有如何在 docker 上部署 deepflow-agent 的内容

**Reply by zcl1115** [reply]
是想问关于【配置 DeepFlow Server】的步骤中，没有对于通过docker compose方式部署的server的操作指引，然后我在docker内部的server执行创建服务域【Host Domain】的时候一直提示can not find controller ip, please specify or create one，不知道应该怎么处理
